evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -223,7 +235,29 @@ def fit(self, X, y=None):
             self._component_obj.fit(y=y)
         return self
 
-    def predict(self, X, y=None):
+    def _manage_types_and_forecast(self, X: pd.DataFrame) -> tuple:
+        fh_ = self._set_forecast(X)
+        X = X.ww.select(exclude=["Datetime"])
+        X.ww.set_types(
+            {
+                col: "Double"
+                for col in X.ww.select(["Boolean"], return_schema=True).columns
+            },
+        )
+        return X, fh_
+
+    @staticmethod
+    def _parse_prediction_intervals(
+        y_pred_intervals: pd.DataFrame,
+        conf_int: int,
+    ) -> Tuple[pd.Series, pd.Series]:
+        preds_lower = y_pred_intervals.loc(axis=1)[("Coverage", conf_int, "lower")]
+        preds_upper = y_pred_intervals.loc(axis=1)[("Coverage", conf_int, "upper")]
+        preds_lower.name = None
+        preds_upper.name = None
```

Comment:
Is this important/needed?

Reply:
Unfortunately indexing the MultiIndex dataframe above 
`y_pred_intervals.loc(axis=1)[("Coverage", conf_int, "lower")]` yields
`(Coverage, 0.95, lower)` as a series name